### PR TITLE
"autoprefixer: ignore next" support

### PR DESCRIPTION
**Pull request by vostrik**
#1024 Added support for comment "autoprefixer: ignore next"

### CODE REVIEW

File: test/cases/ignore-next.out.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  mask: none;
+  background: -webkit-linear-gradient(top, white, black);
+  background: -o-linear-gradient(top, white, black);
+  background: linear-gradient(to bottom, white, black);
+}
+
+/* autoprefixer: ignore next */
+ingore-rule {
+  mask: none;
+  background: linear-gradient(to bottom, white, black);
```
Comment (ai) [inline test/cases/ignore-next.out.css]: We need prefixes here and inside next at-rule. This comment should affect only selector and at-rule params on next node.

Or we will need to think about combining `ignore next` and `off/on`.

File: lib/processor.js [hunk 2]
```diff
@@ -342,6 +342,18 @@ class Processor {
         if (node._autoprefixerDisabled !== undefined) {
             return node._autoprefixerDisabled;
         }
+        if (node.parent && node.parent.nodes) {
+            const nodeIndex = node.parent.nodes.indexOf(node);
+            if (nodeIndex > 0) {
+                const previousNode = node.parent.nodes[nodeIndex - 1];
```
Comment (ai) [inline lib/processor.js]: You can just call `Node#prev` shortcut

Comment (vostrik) [reply lib/processor.js]: Fixed

File: test/cases/ignore-next.out.css (hunk 1 above)
Comment (vostrik) [reply test/cases/ignore-next.out.css]: Fixed.  Disable Autoprefixer only for selector, at-rule or property.

**Pull request merged**